test/e2e/data/cni/calico/calico.yaml

```diff
@@ -0,0 +1,4687 @@
+---
```

Comment:
Could this be downloaded while running test, instead of keeping it in the repo

Reply:
> Could this be downloaded while running test, instead of keeping it in the repo

calico is also used for nested cluster by capi test framework as [ClusterResourceSet](https://cluster-api.sigs.k8s.io/tasks/experimental-features/cluster-resource-set.html) , so we have to save it in a file to satisify [capi test framework](https://github.com/scuzhanglei/cluster-api-provider-virtink/blob/add-e2e-tests/test/e2e/e2e_suite_test.go#L165) as other providers did.